src/styles.scss

```diff
@@ -1,4 +1,6 @@
 body{
     margin: 0;
     padding: 0;
+    background-color: #232323;
+    ;
```

Comment:
do you need this?

Reply:
Yeah... my fault, I have not still configure StyleLint.